tests/Console/Output/DotsOutputTest.php

```diff
@@ -15,24 +15,24 @@
 namespace PhpCsFixer\Tests\Console\Output;
 
 use PhpCsFixer\Console\Output\OutputContext;
-use PhpCsFixer\Console\Output\ProcessOutput;
+use PhpCsFixer\Console\Output\DotsOutput;
 use PhpCsFixer\FixerFileProcessedEvent;
 use PhpCsFixer\Tests\TestCase;
 use Symfony\Component\Console\Output\BufferedOutput;
 
 /**
  * @internal
  *
- * @covers \PhpCsFixer\Console\Output\ProcessOutput
+ * @covers \PhpCsFixer\Console\Output\DotsOutput
  */
-final class ProcessOutputTest extends TestCase
+final class DotsOutputTest extends TestCase
```

Comment:
*idea* 
we could consider to create base test class for XyzOutputTests, to unify what and how do we test.
We used this approach here: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/tests/Console/Report/FixReport/AbstractReporterTestCase.php

Reply:
That's [what I did](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6842/files#diff-005c12ac66ec8494e12d238906f3e1057b2d78a94be8e2665efc29c6b139d305R22) in #6842 where more test-related code is shared between several output writers.

Reply:
would it make sense to have the tests refactoring addressed in refactoring PR and not in introducing new feature PR?